### ISSUE TITLE
CentOS/RHEL 8 installation fails with missing dependency

### ISSUE THREAD

**Issue by SpareSimian** [sbidy/pywizlight#150]
While the pywizlight package is available in EPEL 8, the asyncio-dgram on which it depends has been removed for lack of a maintainer. However, the pywizlight package in EPEL 8 is fairly old and updating it might result in removing the dependency. I'm investigating that. The latest pywizlight installed with pip3.9 as a normal user does work using the python39 package. (I suspect this would also work with python38.)

For now, the RHEL 8 installation instructions in the README don't work. I'll try to create an updated RPM that does.

The dependency that lost its maintainer: https://src.fedoraproject.org/rpms/python-asyncio-dgram

Fedora package page for pywizlight: https://src.fedoraproject.org/rpms/python-pywizlight

**Comment by SpareSimian** [comment]
I'm able to package 0.5.14 on Rocky 8 (clone of RHEL and CentOS 8) using the spec file from the link above, but running it with Python 3.9 fails, unable to find asyncio.run. I suspect some kind of Python library path problem in my installation. 

I needed to install the python3-dataclasses package so that will need to be added to the spec file as an explicit prereq.

**Comment by sbidy** [comment]
Maybe @fabaff can help here. I'm not sure about the current package status.

**Comment by SpareSimian** [comment]
I opened a bugzilla here against the EPEL 8 package: https://bugzilla.redhat.com/show_bug.cgi?id=2132170

Note that this isn't a problem for Fedora because it's a fast-moving distro and has a single recent Python. RHEL 8 and its derivatives are conservative and are still on Python 3.6, but you can install later versions 3.8 and 3.9 in parallel. Fedora doesn't support parallel versions, which is why I suggested in the bug report that the 3.9 requirement be under an RHEL 8-only conditional.

**Comment by fabaff** [comment]
`[asyncio-dgram](https://src.fedoraproject.org/rpms/python-asyncio-dgram)` is no longer needed (https://github.com/sbidy/pywizlight/pull/99) (I corrected it for the NixOS package with https://github.com/NixOS/nixpkgs/pull/198806). Thus, if Fedora has retired that package it's not a big deal for `pywizlight`.

I can try to get `pywizlight` updated for EPEL8 but testing (or everything that goes beyond `mock`) will be an issue.
